plugins/hadoop.yaml

```diff
@@ -0,0 +1,228 @@
+# Plugin Info
+version: 0.0.1
+title: Apache Hadoop
+description: Log parser for Apache Hadoop
+parameters:
+  enable_datanode_logs:
+    label: DataNode Logs
+    description: Enable collection of Hadoop data node logs
+    type: bool
+    default: true
+  datanode_general_log_path:
+    label: DataNode General Log Path
+    description: The absolute path to the data node general logs. Simple globbing can be used to collect multiple log files
```

Comment:
Are we adding the `Simple globbing` note to all our descriptions? Seems like there's been some inconsistency here

Reply:
No. Nice catch. This was how we were doing all of our file path descriptions before in the log bundles. I manually went through and removed these, because I thought it was unnecessary and overly verbose. I just happened to miss this one after generating it.